CHANGELOG.md

```diff
@@ -237,18 +264,27 @@ Fixed
 - Provides better stacktrace in `win_pkg` return. (#56955)
 - Fixed `reg.present` to respect `(Default)` REG_SZ value of an empty string. (#56959)
 - OpenStack driver can now attach to multiple networks, also now respects provided `conn`. (#56960)
+- Fix condition for ConfigTestCase.test_datetime_config_validation (#56968)
```

Comment:
if we're just updating a test case, and not changing code, no changelog entry is necessary.